src/components/map_rotator/index.js

```diff
@@ -35,8 +37,18 @@ var MapRotator = Base.extend({
     }
 
     var heading = - e.geoposition.coords.heading;
-    heading && this.map.el && this.map.el.animate({ rotate: heading + 'deg' }, 1000, 'ease-in-out');
+    var diff = heading-this.last_heading
+    // rotate in shortest direction
+    if(diff>180){
+      heading-=360
+    }
+    else if(diff<-180){
```

Comment:
minor nitpicks:
* Could you add spaces around the parenthesis of the `if` statement and the binary operators? e.g. `<-` looks like it's an operator on its own.
* Use semicolons to and a statement.
* No need for a new line for the else condition.

It would be cool, if you could adjust your code to the general style of this project.

```
if (diff > 180) {
  heading -= 360;
} else if (diff <- 180) {
  heading += 360;
}
```